src/env.py

```diff
@@ -0,0 +1,8 @@
+"""Environment for slot_checker"""
+
+
+SIGNIN_URL = "https://signin.intra.42.fr/users/sign_in"
+PROJECTS_URL = "https://projects.intra.42.fr/projects"
+PROFILE_URL = "https://profile.intra.42.fr"
+
+DEBUG_MODE = "debug_my_slots"
```

Comment:
I think the name can be better with "DEBUG_PROJECT" instead of "DEBUG_MODE". What do you think?

Reply:
agreed